tests/FunderMaps.IntegrationTests/Backend/Application/OrganizationAdminTests.cs

```diff
@@ -25,7 +25,7 @@ public async Task GetOrganizationByIdReturnSingleOrganization()
             // Arrange
             var organization = new OrganizationFaker().Generate();
             var client = _factory
-                .WithAuthentication(options => options.User.Role = ApplicationRole.Administrator)
```

Comment:
Maybe move client creation to the constructor?

Reply:
Complicates things in many cases, prefer not to unless an entire test collection operates on the same client